pkg/odo/cli/pipelines/environment/add.go

```diff
@@ -59,7 +59,7 @@ func (eo *AddEnvParameters) Validate() error {
 func (eo *AddEnvParameters) Run() error {
 	options := pipelines.EnvParameters{
 		EnvName:           eo.envName,
-		PipelinesFilePath: eo.pipelinesFile,
+		PipelinesFilePath: eo.pipelinesFile + "pipelines.yaml",
```

Comment:
Should we append the filename inside Complete()? By doing so we could use the existing TestComplete to cover this case?

Reply:
Chetan , the changes have been addressed kindly review!